roles/ovirt-datacenter-cleanup/README.md

```diff
@@ -4,6 +4,15 @@ oVirt Datacenter Cleanup
 The `ovirt-datacenter-cleanup` role is used to cleanup all entities inside
 oVirt datacenters and finally remove the datacenters themselves.
 
+In the special case that the datacenter provided to cleanup contains a VM that
+serves as Hosted Engine the role will not delete the following entities:
+
+- the hosted engine VM itself
+- the storage domain on which the hosted engine VM resides
```

Comment:
Please add the master storage domain which you have to skip the deletion of it too